share/ci/scripts/install_openexr.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+OPENEXR_VERSION="$1"
+
+git clone https://github.com/openexr/openexr.git
+cd openexr
+
+if [ "$OPENEXR_VERSION" != "latest" ]; then
+    git checkout tags/v${OPENEXR_VERSION} -b v${OPENEXR_VERSION}
+fi
+
+mkdir build
+cd build
+cmake ../.
```

Comment:
To be consistent with Boost & OIIO, cmake should have `-DCMAKE_INSTALL_PREFIX=/usr/local`.